apps/common-app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/types.ts

```diff
@@ -25,22 +25,22 @@ export type SharedValueSnapshot = {
 export type ComponentRef = MutableRefObject<(Component & { props: { style: Record<string, unknown> } }) | null>;
 
 export enum DescribeDecorator {
-  ONLY = 'ONLY',
-  SKIP = 'SKIP',
-  NONE = 'NONE',
+  ONLY = 'only',
+  SKIP = 'skip',
```

Comment:
Why did you remove `NONE`? How is it related to monorepo & eslint?

Reply:
Monorepo required me to use correct types. I wanted to use type `Record<DescribeDecorator, ...>`, so I removed `NONE` which is never used (and now I think I shouldn't have added it to this enum).
Also thats why I changed the values of enum keys - they should match object properties.
Of course I could define more new types without modifying existing ones, but this solution seems the nicest.